packages/azure-http-specs/spec-summary.md

```diff
@@ -180,12 +180,36 @@ Expected response body:
 }
 ```
 
-### Azure_ClientGenerator_Core_HierarchyBuilding_updateDog
+### Azure_ClientGenerator_Core_HierarchyBuilding_AnimalOperations_updateAnimalAsAnimal
 
-- Endpoint: `put /azure/client-generator-core/hierarchy-building/animal/dog`
+- Endpoint: `put /azure/client-generator-core/hierarchy-building/animal/as-animal`
 
 Test operation that accepts Animal input and returns Animal output.
-This operation expects Dog data and validates the @hierarchyBuilding decorator when Dog is used.
+This operation expects Animal data and validates basic Animal functionality.
```

Comment:
I am not sure whether this is valid or not.

There is definition of the `kind` on Animal model.

Reply:
Yeah, `Animal` is most basic class and no default value for "kind". So it is better to remove this scenario.